src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,43 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("d.MM.yyyy");
```

Comment:
Change to proper case
```suggestion
    private static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("d.MM.yyyy");
```